packages/vbet/vbet/vbet.py

```diff
@@ -208,21 +219,24 @@ def vbet(huc, flowlines_orig, flowareas_orig, orig_slope, json_transforms, orig_
 
     # Open evidence rasters concurrently. We're looping over windows so this shouldn't affect
     # memory consumption too much
-    with rasterio.open(proj_slope) as slp_src, \
-            rasterio.open(hand_raster) as hand_src, \
-            rasterio.open(channel_dist_raster) as cdist_src, \
-            rasterio.open(fa_dist_raster) as fadist_src, \
-            rasterio.open(da_slope_zone_raster) as da_slope_zone_src,\
-            rasterio.open(da_hand_zone_raster) as da_hand_zone_src:
+    if in_rasters:
+        open_rasters = {name: rasterio.open(raster) for name, raster in in_rasters.items()}
```

Comment:
If you're going to do this (totally valid by the way) you need to explicitly close them later or the file handles will be left open.

The .close() is implicit when using the "with" statement

https://rasterio.readthedocs.io/en/latest/topics/reading.html